source/_integrations/transmission.markdown

```diff
@@ -143,6 +144,25 @@ Stops a torrent.
 | `entry_id`             | no       | The integration entry_id                                                                    |
 | `id`                   | no       | ID of the torrent, can be found in the `torrent_info` attribute of the `*_torrents` sensors |
 
+### Action `get_torrents`
+
+Gets torrent details for the selected filter.
+
+This action populates [Response Data](/docs/scripts/perform-actions#use-templates-to-handle-response-data) with a dictionary of torrents based on the provided filter.
+
+| Data attribute         | Optional | Description                                                                                |
```

Comment:
I know this is the style that is used in the rest of the document, and what you are doing is consistent with the rest.. 
But we are trying to move towards the style used in this example here:
https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L139

- the title uses the UI string, not the YAML. the YAML is then used in the intro.
- The descriptive content is not in a table but uses a list instead.